assets/src/edit-story/components/panels/design/link/link.js

```diff
@@ -52,6 +52,11 @@ const IconText = styled(Text)`
   color: ${({ theme }) => theme.colors.fg.secondary};
 `;
 
+const MediaWrapper = styled.div`
+  width: 54px;
+  height: 54px;
+`;
```

Comment:
Could you not just do `const StyledMedia = styled(Media)``;` and let it accept either `...rest` or a `className`?